packages/rocketchat-ui-sidenav/client/roomList.js

```diff
@@ -73,6 +73,15 @@ Template.roomList.helpers({
 	}
 });
 
+Template.roomList.rendered = function() {
+	if (this.data.identifier !== undefined && this.data.identifier === 'c') {
+		const _first_child = this.find('.channelItemLists').children[0];
+		if (_first_child !== '' && _first_child !== undefined) {
+			_first_child.children[0].click();
```

Comment:
So I'll add one textbox in Admin ~~ General section with label "First Channel After Login". User can provide the channel he want. If no channel is provided or invalid channel is provided then default home page will be loaded.

will this be fine?